style.css

```diff
@@ -1,2 +1,364 @@
 
 
+ /* FONTS  */
+
+/* ---OSWALD---- */
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+/* ---- OPEN SANS ---- */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&display=swap');
+/* ---- ROBOTO---- */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+/* --- Raleway --- */
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;500;600;700;800;900&display=swap');
+/* INTER VAN */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800;900&display=swap');
+
+/* ---- FONT AWESOME ---- */
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css');
+
+
+:root{
+        /* FONT SIZE  */
+
+        font-size:62.5%;
+
+        /* COLORS */
+    
+        --COLOR-PRIMARY:#44d62c;
+        --COLOR-LIGHT-PRIMARY:#73e161;
+        --COLOR-DARK-PRIMARY:#1dd800;
+
```

Comment:
Awesome. The root is loaded with CSS variables to ease out. Codes are reliable and arranged in nature.